app/views/pieces/show.html.haml

```diff
@@ -9,11 +9,28 @@
 %div= render 'reviews/form'
 
 - if !@reviews.nil?
-  %ul
+  %ul.clearfix#review_block
     - @reviews.each do |review|
       -if !review.title.empty?
-        %li= review.title
-        %li= review.body
-        %li= "written by: #{review.user.username}"
+        %div.review-container{id: "review_id-#{review.id}"}
+          %div.voting-title-container.clearfix
+            %div.voting{review_id: "#{review.id}"}
+              - if user_signed_in?
+                - current_vote = current_user.votes.where(review: review).first
+              - if current_vote.nil? || current_vote.up == 0
+                %span.nil_vote{change_up_to: "1"}= link_to "+", votes_path(vote: {review_id: review.id, up: 1}), method: :post
+                %span.nil_vote{change_up_to: "-1"}= link_to "-", votes_path(vote: {review_id: review.id, up: -1}), method: :post
```

Comment:
Line is too long. [128/80]<br>Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace